gdnative-derive/src/lib.rs

```diff
@@ -164,6 +197,44 @@ pub fn profiled(meta: TokenStream, input: TokenStream) -> TokenStream {
 /// - `no_editor`
 ///
 /// Hides the property from the editor. Does not prevent it from being sent over network or saved in storage.
+///
+/// ### `#[methods]`
+/// Adds the necessary information to an `impl` block for 
+///
+/// ### `#[export]`
+/// Registers the attributed function signature to be used by Godot.
+/// A valid function signature must 
+/// - `&self` or `&mut self` as its first argument
+/// - `&<T>` or `TRef<T>` where T refers to the type declared in `#[inherit(T)]` attribute as it's second argument.
+/// - Additional arguments must implement the `FromVariant` trait or be a Variant type.
+/// - Return values must implement the ToVariant trait or be a Variant type.
+/// 
+/// ```ignore
+/// #[export]
+/// fn foo(&self, owner: &Reference)
+/// ```
+/// **Note**: Marking a fn block with `#[export]` only is only supported inside an `impl` block that has the `#[methods]` attribute.
+///
+/// Possible arguments for this attribute are
+///
+/// - `name` = "overriden_function_name"
+///
+/// Overrides the function name as the method name to be registered in for godot.
+///
+/// - `rpc` = "selected_rpc"
+///   - "selected_rpc" must be one of the values be one of the following which refer to the associated [Multiplayer API RPC Mode](https://docs.godotengine.org/en/stable/classes/class_multiplayerapi.html?highlight=RPC#enumerations)
+///     - "disabled" - RPCMode::RPC_MODE_DISABLED
+///     - "remote" - RPCMode::RPC_MODE_REMOTE 
+///     - "remote_sync" - RPCMode::RPC_MODE_REMOTE_SYMC
+///     - "master" - RPCMode::RPC_MODE_MASTER
+///     - "puppet" - RPCMode::RPC_MODE_PUPPET
+///     - "master_sync" - RPCMode::RPC_MODE_MASTERSYNC
+///     - "puppet_sync" - RPCMode::RPC_MODE_PUPPETSYNC
```

Comment:
"must be one of the values be one of the following which"
-> "must be one of the following values, which"
(remove 2x, add comma)

Would it be possible to add backticks around `RPCMode` constants?

Reply:
Done :)